pkgs/development/interpreters/python/default.nix

```diff
@@ -13,6 +13,12 @@
 
 (let
 
+  checkPythonPkgsAttrNames = self: super:
```

Comment:
Good to have this check!
(just putting this comment here so it is easier to find this function when reviewing)